Python/Product/Cookiecutter/source.extension.vsixmanifest

```diff
@@ -9,7 +9,7 @@
     <Icon>Resources\Cookiecutter_32x.png</Icon>
     <PreviewImage>Resources\CookiecutterTemplate_200x.png</PreviewImage>
   </Metadata>
-  <Installation SystemComponent="true" AllUsers="true" InstalledByMsi="true">
+  <Installation SystemComponent="false" AllUsers="true" InstalledByMsi="false">
```

Comment:
AllUsers="false" seemed to work when I created a VSIX for Jude, can we go with that?

Reply:
This would change historic behavior though

Reply:
Well, installing from vsix has not been possible in the past since there hasn't been a vsix.

Not requiring elevation when installing seems beneficial.

Reply:
Sure, I don't mind.